parachain/pallets/basic-channel/src/outbound/mod.rs

```diff
@@ -121,6 +75,10 @@ pub mod pallet {
 
 		type Hashing: Hash<Output = H256>;
 
+		/// ID of message source
+		/// + TypeInfo,
+		type SourceId: Parameter + Member + MaxEncodedLen;
```

Comment:
Should also be bounded by `PartialEq`. I know that's already defined by Member, but it is good to repeat it here for documentation purposes